src/md.cc

```diff
@@ -428,7 +443,14 @@ void MGmol<OrbitalsType>::md(OrbitalsType** orbitals, Ions& ions)
         bool last_move_is_small = true;
         do
         {
-            retval = quench(**orbitals, ions, ct.max_electronic_steps, 0, eks);
+            if (ct.getROMOptions().rom_stage == ROMStage::TEST_ORBITAL)
+            {
+                updateDMandEnergy(**orbitals, ions, eks);
```

Comment:
Do we need a check for `retval` as in quench? I think this is not supported in the MVP solver yet.

Reply:
No. updateDMandEnergy() does not return anything.